daisy_workflows/export/disk_export_ext.wf.json

```diff
@@ -32,37 +32,50 @@
     }
   },
   "Sources": {
-    "${NAME}_export_disk_ext.sh": "./export_disk_ext.sh"
+    "${NAME}_export_disk_ext.sh": "./export_disk_ext.sh",
+    "disk_resizing_mon.sh": "./disk_resizing_mon.sh"
   },
   "Steps": {
     "setup-disks": {
       "CreateDisks": [
         {
-          "Name": "disk-${NAME}",
+          "Name": "disk-${NAME}-os-${ID}",
           "SourceImage": "${export_instance_disk_image}",
-          "SizeGb": "${export_instance_disk_size}",
-          "Type": "pd-ssd"
+          "Type": "pd-ssd",
+	  "ExactName": true
+        },
+        {
+          "Name": "disk-${NAME}-buffer-${ID}",
+          "SizeGb": "${buffer_disk_size}",
+          "Type": "pd-ssd",
+	  "ExactName": true
         }
       ]
     },
     "run-${NAME}": {
       "CreateInstances": [
         {
           "Name": "inst-${NAME}",
-          "Disks": [{"Source": "disk-${NAME}"}, {"Source": "${source_disk}", "Mode": "READ_ONLY"}],
+          "Disks": [
+            {"Source": "disk-${NAME}-os-${ID}"},
+            {"Source": "${source_disk}", "Mode": "READ_ONLY"},
+            {"Source": "disk-${NAME}-buffer-${ID}"}
+          ],
           "MachineType": "n1-highcpu-4",
           "Metadata": {
             "block-project-ssh-keys": "true",
             "gcs-path": "${OUTSPATH}/${NAME}",
-            "format": "${format}"
+            "format": "${format}",
+	    "instance-name": "inst-${NAME}",
+            "buffer-disk": "disk-${NAME}-buffer-${ID}"
           },
           "networkInterfaces": [
             {
               "network": "${export_network}",
               "subnetwork": "${export_subnet}"
             }
           ],
-          "Scopes": ["https://www.googleapis.com/auth/devstorage.full_control"],
+          "Scopes": ["https://www.googleapis.com/auth/devstorage.full_control", "https://www.googleapis.com/auth/cloud-platform"],
```

Comment:
cloud-platform scope is too broad, all you should need is https://www.googleapis.com/auth/compute